cluster-api-gcp/CONTRIBUTING.md

```diff
@@ -23,10 +23,15 @@ Steps to follow:
 2.  Configure Cloud SDK to point to the GCP project you will be using.
 
     ```bash
-    $ gcloud auth login
     $ gcloud config set project <GCP_PROJECT_ID>
     ```
 
+3.  Set the client credentials to be used by Cluster API to create resources.
```

Comment:
Personally, I liked the fact that the original instructions talked about GOOGLE_APPLICATION_CREDENTIALS and what the creds are used for. As a contributor, it is good to know what I am doing.

Reply:
I added it back and have both in the file now.